custom-instrumentation/instrument-datastore/index.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2024 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+
+const { SimpleDatastore } = require('./simple-datastore')
+const newrelic = require('newrelic')
```

Comment:
While `newrelic` is already loaded in such a fashion that it will instrument things, via `-r newrelic`, it would be best to swap the ordering here to prevent copy and paste errors:

```suggestion
const newrelic = require('newrelic')
const { SimpleDatastore } = require('./simple-datastore')
```